.gitignore

```diff
@@ -9,4 +9,5 @@ packages/*/out
 packages/*/docs/www
 lerna-debug.log
 .idea/
-.vscode/
\ No newline at end of file
+.vscode/
+*.orig
```

Comment:
what is `.orig`?

Reply:
This is for local development using Vim. When modifying a file, say `dev.js,` Vim keeps a local history of that file's changes in `dev.js.orig`. There are other ways to avoid committing .orig files, but this line is a failsafe.